src/test/java/com/shippo/model/ManifestTest.java

```diff
@@ -78,6 +78,7 @@ public static Object getDefaultObject() throws AuthenticationException, InvalidR
         Address testAddress = (Address) AddressTest.getDefaultObject();
         Transaction transaction = (Transaction) TransactionTest.getDefaultObject();
         List<String> transactions = new ArrayList<String>();
+        transactions.add(transaction.objectId);
```

Comment:
We weren't adding the transaction to the transactions list, so the API would grab all transactions from this carrier for the last day. Flakiness cause #1